src/foam/mlang/mlang.js

```diff
@@ -564,7 +564,7 @@ foam.CLASS({
           sortedArgs[costKey] = arg;
           costs.push(costKey);
         }
-        costs = costs.sort(foam.Number.compare);
+        costs = costs.sort(foam.util.compare);
```

Comment:
In hindsight, I think we don't need to change this. using `foam.<type>.compare(a, b)` directly is safe iff `a` and `b` are known to be of type `<type>`. That is the case here. It's also the case [here](https://github.com/foam-framework/foam2/blob/2868ed4916cc7a3477708e7538f7a4e27421b888/src/foam/core/types.js#L101).